deltachat-ffi/src/lib.rs

```diff
@@ -225,17 +225,19 @@ pub unsafe extern "C" fn dc_get_info(context: *const dc_context_t) -> *mut libc:
     }
     let ctx = &*context;
     block_on(async move {
-        let info = ctx.get_info().await;
+        let info = ctx.get_info().await.log_err(ctx, "failed to get info").ok();
         render_info(info).unwrap_or_default().strdup()
     })
 }
 
 fn render_info(
-    info: BTreeMap<&'static str, String>,
+    info: Option<BTreeMap<&'static str, String>>,
```

Comment:
Maybe just return `"".strdup()` in `dc_get_info` in case of error and keep `render_info` as is?

Reply:
Pushed the change